### PR TITLE
Support passing a custom coverage formatter to finalizeCoverage()

### PR DESCRIPTION
**Pull request by cbracken**

### CODE REVIEW

File: packages/flutter_tools/lib/src/test/coverage_collector.dart [hunk 1]
```diff
@@ -58,16 +58,18 @@ class CoverageCollector {
   List<Future<Null>> _jobs = <Future<Null>>[];
   Map<String, dynamic> _globalHitmap;
 
-  Future<String> finalizeCoverage() async {
+  Future<String> finalizeCoverage({Formatter formatter}) async {
```
Comment (abarth) [inline packages/flutter_tools/lib/src/test/coverage_collector.dart]: We put spaces inside the `{` and `}` because we're unique snowflakes.

**Comment by abarth** [comment]
LGTM

File: packages/flutter_tools/lib/src/test/coverage_collector.dart (hunk 1 above)
Comment (Hixie) [inline packages/flutter_tools/lib/src/test/coverage_collector.dart]: _pretty_ snowflakes. Because we're unique _pretty_ snowflakes.

Comment (cbracken) [inline packages/flutter_tools/lib/src/test/coverage_collector.dart]: And in my case, apparently illiterate! Missed this comment; will send a quick patch :)

**Pull request merged**